src/Applications/conf.yaml

```diff
@@ -33,13 +35,14 @@
 work_model:
   name: "AffineCombination"
   parameters:
-    alpha: 1.
-    beta: 0.
+    alpha: 0.
```

Comment:
Do you want to commit the change to alpha/beta here?

Reply:
Thanks for catching this! 

I restored the default params to their previous values.